webapp/package.json

```diff
@@ -1,11 +1,11 @@
 {
-  "name": "medic-webapp",
+  "name": "medic",
```

Comment:
You've changed the name in `webapp/package.json` but not `webapp/package-lock.json` - I think these two need to be consistent. The name is a bit weird because we don't publish webapp separately. Either `medic-webapp` or `medic/webapp` or `@medic/webapp` or just simply `webapp` would be fine I think.

For reference, `/admin/package.json` has the name `medic-admin`.